src/handlers/messages.ts

```diff
@@ -324,6 +380,58 @@ export async function handle(rcMessage: RcMessage): Promise<void> {
     return
   }
 
+  const ts = new Date(rcMessage.ts.$date).valueOf()
+  if (rcMessage.file) {
+    if (rcMessage.attachments?.length == 1) {
+      const path = './inputs/files/' + rcMessage.file._id
```

Comment:
Could you please create an empty `inputs/files/.gitkeep`, like we have in `inputs/`?

Reply:
Yes, done.